### DISCUSSION TITLE
Updating via Web

### DISCUSSION THREAD

**Discussion by SpeedyCharly** [AzuraCast/AzuraCast#6086] [General]
Am I the only one or does someone else have trouble updating via the new Web button?  Tried a few times but got error message that service wasn't accessible.  Proceeded to update via Putty as usual, then returned to the Web interface.  No error on that second try, but then there was nothing left to update...

Is there a system-wide glitch - or is it just me?

**Comment by LordFader** [comment]
1st time I see that option it works a few months ago... but after some updates it doesn't work anymore... it show error of something.

**Comment by BusterNeece** [comment]
Timeout has been increased on the web updater so it shouldn't be throwing as many confusing errors.

**Reply by LordFader** [reply]
Rolling Release #[6d81fae]• Docker • PHP 8.2

Well... I just update thru Terminal...
But I run the Web Update and don't return the red error.

On the next update I will let you know.

thx for your time.

**Comment by SpeedyCharly** [comment]
will test and report tonite... Thanks Buster

**Comment by SpeedyCharly** [comment]
Was 9 commits behind; clicked get updates = all fine.
Clicked Update Via Web = same error as before.  Clicked again and update proceeded fine.

Mebbe timeout needs to be a hair longer still...

**Reply by LordFader** [reply]
I tried a update via Web right now, and:

An error occurred and your request could not be completed.

And all stop and 502 Bad Gateway Error

... after force a reboot:
Rolling Release #[919e3aa] (2023-02-21 6:29) • Docker • PHP 8.2

It looks like it did the update...
But not smooth like in the beginning of this option show up.

**Reply by LordFader** [reply]
Any log you need?
I don't know which one you need and how... (well if it's not in admin/logs)

**Comment by LordFader** [comment]
[AzuraCast] • Rolling Release #[919e3aa] • Docker • PHP 8.2

![image](https://user-images.githubusercontent.com/4002168/221354993-a7218505-a522-409f-b3e9-3013f074283d.png)

Going for manual update.
I will give feedback in the next update.

**Comment by Moonbase59** [comment]
I tend to use the CLI update method, but I felt brave today and tried the Web Update (to Rolling Release #[5cbf9c9](https://github.com/AzuraCast/AzuraCast/commit/5cbf9c91cae1a234c428cf43d769fe2349e05bd9) (2023-02-28 8:15) from the one before).

Apart from the error message, all went beautifully well, one only has to wait long enough and log in again, of course. The error probably stems from the system "going away" due to the update. Not a problem for me.

For an end-user, it might be good to jump to an empty page stating AzuraCast is updating, and to please be patient, with maybe a "Re-Login" button that would jump to the login page?
